plugins/permission-common/src/PermissionClient.test.ts

```diff
@@ -211,7 +218,284 @@ describe('PermissionClient', () => {
     });
   });
 
+  describe('authorize (batched)', () => {
+    beforeAll(() => {
+      client = new PermissionClient({
+        discovery,
+        config: new ConfigReader({
+          permission: {
+            enabled: true,
+            EXPERIMENTAL_enableBatchedRequests: true,
+          },
+        }),
+      });
+    });
+
+    const mockAuthorizeConditional = {
+      permission: mockPermission,
+      resourceRef: 'foo:bar',
```

Comment:
maybe doesn't matter for the test but these (here and elsewhere) really should always be full refs